toolset/setup/linux/prerequisites.sh

```diff
@@ -42,7 +42,8 @@ sudo apt-get -qqy install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options:
   libpq-dev mlton \
   libjemalloc-dev libluajit-5.1-dev `# Needed by lwan at least` \
   libhiredis-dev                    `# Redis client - Needed by ngx_mruby at least` \
-  cloc dstat                        `# Collect resource usage statistics`
+  cloc dstat                        `# Collect resource usage statistics` \
+  libsasl2-dev                      `# Needed by mgo for go-mongodb test`
```

Comment:
Thought about it a bit and I guess there is no better way to do this at the moment - and you're right that it's important enough it may be a common prereq in the future. Mind changed - PR LGTM